packages/auth/src/access/model.ts

```diff
@@ -28,3 +32,15 @@ export class Access extends BaseModel {
   public identifier?: string
   public limits?: LimitData
 }
+
+export function toOpenPaymentsAccess(
+  accessItem: Access
+): OpenPaymentsAccessItem {
+  return {
+    actions: accessItem.actions,
+    identifier: accessItem.identifier as string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    type: accessItem.type as any,
+    limits: accessItem.limits ?? undefined
```

Comment:
Should we include this?

Reply:
Yes. If an outgoing payment access has `limits` it should be included.